### PR TITLE
RMC-134 Large Print Questionnaires and Translation Booklets

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
RM will now generate supplementary printed materials for fulfilment requests, this PR adds acceptance tests for some examples of large print questionnaires and translation booklets. 

# What has changed
* Add scenario outline for supplementary printed material fulfilment requests
* Remove code which kept retrying print file matching even after a file has been found and doesn't match (now just fails the test immediately)

# How to test?
Build and run the linked action scheduler and print file service branches, then run these acceptance tests

# Links
https://trello.com/c/yDd9LhBR/1009-rmc-134c2-implement-large-print-and-translations-requested-event-5
https://github.com/ONSdigital/census-rm-action-scheduler/pull/27
https://github.com/ONSdigital/census-rm-print-file-service/pull/10

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[RMC-134(C2) Implement Large Print and Translations requested event (5)](https://trello.com/c/yDd9LhBR/1009-rmc-134c2-implement-large-print-and-translations-requested-event-5)

File: acceptance_tests/features/steps/fulfilment_request.py [hunk 1]
```diff
@@ -42,8 +46,7 @@ def send_fulfilment_requested_event(context, pack_code):
             }
         }
     )
-
-    time.sleep(2)
+    # time.sleep(2)
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/fulfilment_request.py]: Can you remove this now?

File: acceptance_tests/utilities/mappings.py [hunk 2]
```diff
@@ -31,6 +60,34 @@
...
+    "P_TB_TBPOL1": PPD1_3_DATASET,
+    "P_TB_TBPOL4": PPD1_3_DATASET,
+    "P_TB_TBPOR1": PPD1_3_DATASET,
+    "P_TB_TBRUS1": PPD1_3_DATASET,
+    "P_TB_TBSOM1": PPD1_3_DATASET,
+    "P_TB_TBSPA1": PPD1_3_DATASET,
+    "P_TB_TBTUR1": PPD1_3_DATASET,
+    "P_TB_TBULS4": PPD1_3_DATASET,
+    "P_TB_TBVIE1": PPD1_3_DATASET,
+    "P_TB_TBYSH1": PPD1_3_DATASET,
```
Comment (ryangrundy7) [inline acceptance_tests/utilities/mappings.py:46]: Missing `P_TB_TBURD1 ` from this mapping.

File: acceptance_tests/utilities/mappings.py [hunk 3]
```diff
@@ -18,6 +19,34 @@
...
+    "P_TB_TBPOL1": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBPOL4": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBPOR1": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBRUS1": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBSOM1": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBSPA1": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBTUR1": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBULS4": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBVIE1": Config.SFTP_PPO_DIRECTORY,
+    "P_TB_TBYSH1": Config.SFTP_PPO_DIRECTORY,
```
Comment (ryangrundy7) [inline acceptance_tests/utilities/mappings.py:27]: Missing `P_TB_TBURD1` from this mapping

**Review by ryangrundy7 — COMMENTED** [review]
Tests all pass locally. Just some comments to address

File: acceptance_tests/utilities/mappings.py (hunk 2 above)
Comment (AdamHawtin) [reply acceptance_tests/utilities/mappings.py:46]: I'm going to delete all the unused mappings

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

**Review by liamtoozer — APPROVED** [review]
LGTM 👍 Tested locally

**Pull request merged**